src/index.jsx

```diff
@@ -61,25 +42,25 @@ class App extends Component {
 
   render() {
     return (
-      <AuthStore repo={this.authRepo}>
-        <div className="app">
-          <Header/>
-          <Main/>
-          <Footer/>
-        </div>
-      </AuthStore>
+      <AuthContext repo={this.authRepo}>
+        <ApolloContext>
```

Comment:
In a perfect world maybe we could get rid of authContext as a context, and make it a higher ordered component that returns a handle to the token that would then get wired into apollo's context as a prop.

Reply:
> that returns a handle to the token

I'm down with an HOC, but I don't know what this part means.

Reply:
i just meant that would be what the HOC provides to its children -- the auth token, vs. providing it in the context. its an optimization for a later date tho.